tests/Browser/BlogManageTest.php

```diff
@@ -0,0 +1,562 @@
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->assertDontSee('Delete');
+        });
+    }
+
+    /**
+     * Test Enter url not defined in route
+     * case 1.7: Enter url not defined in route
+     *
+     * @return void
+     */
+    public function testLoginUrlNotExist()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit('/urlkhongtontai')
+                    ->assertSee('404');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.1: Create blog: Fill all input
+     *
+     * @return void
+     */
+    public function testCreateBlogFillAllInput()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', 'Lốc bụi cao 650 m trên sao Hỏa')
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
+                    
+            $browser->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('Create new blog success !')
+                    ->assertSee('Lốc bụi cao 650 m trên sao Hỏa');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.2: Create blog: Fill all input but fill in title less than 5 characters
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogFillTitleLessThan5Character()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', 'Lốc')
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
+
+            $browser->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title must be at least 5 characters.')
+                    ->assertDontSee('Lốc bụi');
```

Comment:
how can a input 'Lốc' expected to see 'Lốc bụi' ?

Reply:
I fixed